src/components/UpdateStatusModal/index.tsx

```diff
@@ -0,0 +1,177 @@
+          className="margin-top-neg-2 margin-bottom-2"
+        >
+          {modelPlanTaskListT('statusModal.heading')}
+        </PageHeading>
+
+        <p className="font-body-md line-height-sans-4 margin-top-0 margin-bottom-2">
+          {modelPlanTaskListT(`statusModal.statusText.${suggestedPhase.phase}`)}
+        </p>
+
+        <p className="font-body-md line-height-sans-4 margin-top-0 margin-bottom-2 text-bold">
+          {modelPlanTaskListT('statusModal.currentStatus')}
+          <span className="text-normal">
+            {statusConfig.options[currentStatus]}
+          </span>
+        </p>
+
+        <p className="font-body-md line-height-sans-4 margin-top-0 margin-bottom-2 text-bold">
+          {suggestedPhase.phase !== ModelPhase.IN_CLEARANCE ? (
+            <span className="display-flex">
+              {modelPlanTaskListT('statusModal.newStatus')}
+              <span className="margin-right-05">: </span>
+              <span className="text-normal">
+                {statusConfig.options[suggestedPhase.suggestedStatuses[0]]}
+              </span>
+            </span>
+          ) : (
+            <>
+              {modelPlanTaskListT('statusModal.newStatus')}
+              <div className="text-normal text-base">
+                {modelPlanTaskListT('statusModal.hint')}
+              </div>
+              <Select
+                id="sort"
+                className="margin-bottom-2 margin-top-1"
+                name="status"
+                value={status}
+                onChange={(e: ChangeEvent<HTMLSelectElement>) => {
+                  setStatus(e.target.value as ModelStatus);
+                }}
+              >
+                {getKeys(statusConfig.options)
+                  .filter(statusOption =>
+                    suggestedPhase.suggestedStatuses.includes(statusOption)
+                  )
+                  .map(statusOption => {
+                    return (
+                      <option
+                        key={`Status-Dropdown-${statusConfig.options[statusOption]})}`}
+                        value={statusOption}
+                      >
+                        {statusConfig.options[statusOption]}
+                      </option>
+                    );
+                  })}
+              </Select>
+            </>
+          )}
+        </p>
+
+        <div className="margin-top-4 margin-bottom-1">
+          <Button
+            type="button"
+            data-testid="update-status"
+            onClick={() => {
+              handleUpdateStatus();
+            }}
+          >
+            {modelPlanTaskListT('statusModal.update')}
+          </Button>
+
+          <Button
+            type="button"
+            className="margin-left-2 text-red"
+            data-testid="go-to-timeline"
+            unstyled
+            onClick={() => {
+              closeModal();
+            }}
+          >
+            {modelPlanTaskListT('statusModal.goToTimeline')}
+          </Button>
```

Comment:
Figma has the link in blue and, according to Zoe's comment, go to the timeline 

![image](https://github.com/user-attachments/assets/b47d109e-2401-4d88-a1a0-8758c02a3a35)

Reply:
Ahh I was reading it as task list lol, nice catch